examples/media/src/search.ts

```diff
@@ -21,7 +21,7 @@ const searchClient = algoliasearch(
 const search = instantsearch({
   searchClient,
   indexName: 'PROD_algolia_blog',
-  routing: true,
+  routing: { stateMapping: singleIndex('PROD_algolia_blog') },
```

Comment:
sorry for not mentioning in the earlier suggestion, but this needs to be imported still

Reply:
no worries, i added it in c8de92a.